apps/chat/src/store/conversations/conversations.selectors.ts

```diff
@@ -740,3 +740,51 @@ export const selectIsConversationsEmpty = createSelector(
     });
   },
 );
+
+export const selectIsSelectMode = createSelector([rootSelector], (state) => {
+  return (
+    state.chosenConversationIds.length > 0 || state.chosenFolderIds.length > 0
+  );
+});
+
+export const selectChosenConversationIds = createSelector(
+  [rootSelector],
+  (state) => {
+    return state.chosenConversationIds;
+  },
+);
+
+export const selectChosenFolderIds = createSelector([rootSelector], (state) => {
+  return state.chosenFolderIds;
+});
+
+export const selectAllChosenFolderIds = createSelector(
```

Comment:
`selectChosenFolderIds` isn't enough?

Reply:
no, we have chosen folders, but have child folders which selected because parent is chosen